docs/tutorial.rst

```diff
@@ -263,7 +263,7 @@ Next connect to node B and do the same process. We'll do both steps at once:
        --ssl-self-signed \
        --username ha-admin \
        --dbname appdb \
-       --nodename ha-demo-b.internal.cloudapp.net \
+       --hostname ha-demo-b.internal.cloudapp.net \
        --pgctl /usr/lib/postgresql/11/bin/pg_ctl \
        --monitor 'postgres://autoctl_node@ha-demo-monitor.internal.cloudapp.net/pg_auto_failover?sslmode=require'
```

Comment:
should we add a `--name` argument to these nodes as well?

Reply:
I think it's good to show the defaults. In a later version of the tutorial, when we use multiple zones, I will given them names like east-a and west-c and things.